src/renderer/utils/InstallerConfiguration.ts

```diff
@@ -52,7 +52,7 @@ export class InstallerConfiguration {
                     } else {
                         return Promise.reject('Both network and local configurations are invalid');
                     }
-                });
+                }).catch((error) => Promise.reject(new Error(error.message)));
```

Comment:
Just reject the Error and don't catch it and reject again.

Reply:
fixd